docs/source/author/content_style_guide.rst

```diff
@@ -3,3 +3,32 @@ Content Style Guide
 
 .. contents:: Contents
   :local:
+
+Writing Style
+==============================================================================
+
+When using pronouns in reference to a hypothetical person, gender neutral pronouns (they/their/them) should be used.
+
+All documents (other than those for internal use only) must be written clearly and simply so that a non-expert is able to understand them.
+Preferably documents should be readable by students.
+
+Any jargon used needs to be clearly explained and should be considered as a glossary definition.
+
+Capitalisation Rules
+------------------------------------------------------------------------------
+In the majority of cases capitalisation should not be used for keywords and titles.
+
+The following wouldn't be capitalised:
+binary number(s), digits, binary digits
```

Comment:
It's pretty easy to tell something else *used* to be in here, perhaps say 'For example, the following wouldn't...`

Tim used to say follow how Wikipedia does capitalisation, but we can't really say that here can we?